webpack.config.js

```diff
@@ -85,12 +85,6 @@ module.exports = (env, argv) => {
       libraryTarget: 'umd',
       globalObject: 'this'
     },
-    resolve: {
```

Comment:
@nickw1 i think this is not needed for aframe-location-only dist lib, tell me if i'm right.

Reply:
@kalwalt will look at this in the next couple of days, sorry have had a busy week!

Reply:
@nickw1 no problem! The code should be ok if you can test or just take a look at, it would be nice. :slightly_smiling_face:

Reply:
@kalwalt done! Sorry for the delay, was difficult to find the time last week.

Reply:
ok so i will publish a new npm package!